### PR TITLE
Issues/745 remove stale code

### PR DESCRIPTION
**Pull request by reusr1**
this PR removes some unused bundles and brings peregrine up to the latest versions of the corresponding 3rd party bundles.
issues/707 branch is also merged into this PR (new travis build)

please review and merge https://github.com/headwirecom/peregrine-cms/pull/708 (issues/707 PR) before reviewing this PR

### CODE REVIEW

**Review by cmrockwell — APPROVED** [review]
Seems the bulk of this PR happens in commits: afb18edebfaa51570d9030892369247a656d5612, 75bba04999f93d90c8afbc90ea2be97f62342ace and f47b482e421de4fd83637c78f834eaa7ee61a5ce 
with the rest of the commits coming from PR #708 

On my system, some artifacts of the removed modules remained. 
I cleaned it up like this.
rm -rf platform/node-js
rm -rf samples/marketplace
rm -rf sling/peregrine-builder
rm -rf sling/peregrine-feature-model-starter

Regards

File: platform/base/ui.apps/src/main/content/jcr_root/apps/runmodes/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-peregrine.config [hunk 1]
```diff
@@ -62,9 +62,9 @@ scripts=[\
 # Set Permissions for Sling Distribution
     set ACL for distribution-agent-user, defaultAgentService
         allow jcr:all on /content
-        allow jcr:all on /etc/distribution
-        allow jcr:all on /libs/sling/distribution
-        allow jcr:all on /var/sling/distribution
+#        allow jcr:all on /etc/distribution
+#        allow jcr:all on /libs/sling/distribution
+#        allow jcr:all on /var/sling/distribution
```
Comment (reggie7) [inline platform/base/ui.apps/src/main/content/jcr_root/apps/runmodes/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-peregrine.config:67]: I think we should revert this as I guess it provides a different fix from what we have in #757. @cmrockwell you are probably the most interested in `distribution` replication type and I see your approve here - should we maybe just remove the paths created here or rather go with #757?

File: pom.xml [hunk 2]
```diff
@@ -632,12 +632,12 @@
...
             </dependency>
 
-            <dependency>
+            <!-- <dependency>
                 <groupId>com.amazonaws</groupId>
                 <artifactId>aws-java-sdk-osgi</artifactId>
                 <version>${aws-java-sdk-osgi.version}</version>
                 <scope>provided</scope>
-            </dependency>
+            </dependency> -->
```
Comment (reggie7) [inline pom.xml]: Let's just remove this dependency instead of commenting it out maybe?

**Review by reggie7 — CHANGES_REQUESTED** [review]

File: platform/base/ui.apps/src/main/content/jcr_root/apps/runmodes/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-peregrine.config (hunk 1 above)
Comment (cmrockwell) [reply platform/base/ui.apps/src/main/content/jcr_root/apps/runmodes/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-peregrine.config:67]: My approval here was a mistake. I went ahead and approved #757 but that only creates the paths so there are not errors when setting ACL's. With these ACL's removed. remote replication is broken.

**Pull request merged**